baseapp_payments/rest_framework/customers/serializers.py

```
@@ -0,0 +1,78 @@
+        try:
+            stripe_customer = StripeService().create_customer(email=user.email)
+            if isinstance(stripe_customer, dict) and "id" in stripe_customer:
+                customer = Customer.objects.create(
+                    entity=entity, remote_customer_id=stripe_customer.get("id")
+                )
+            else:
+                customer = Customer.objects.create(
+                    entity=entity, remote_customer_id=stripe_customer.id
+                )
+        except IntegrityError:
+            raise serializers.ValidationError("Customer already exists for this entity.")
```

Ruff (0.15.7)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)